Doppler.BigQueryMicroservice.Test/BigQueryTest.cs

```diff
@@ -212,5 +216,59 @@ public async Task PUT_save_allowed_emails_user_found()
 
         }
 
+        [Fact]
+        public async Task POST_sendWithTemplateAsync()
+        {
+            // Arrange
+            var fixture = new Fixture();
+            var apiKey = fixture.Create<string>();
+            var relayAccountId = fixture.Create<int>();
+            var relayAccountName = fixture.Create<string>();
+            var relayUserEmail = "salesrelay@dopplerrelay.com";
+            var templateId = fixture.Create<string>();
+            var demoData = fixture.Create<string>();
+            var toEmail = "email@gmail.com";
+            var bccEmail = "copy@copy.com";
+            var expectedUrl = $"https://api.dopplerrelay.com/accounts/{relayAccountId}/templates/{templateId}/message";
+
+            var configuration = new RelayEmailSenderConfiguration()
+            {
+                SendTemplateUrlTemplate = SendTemplateUrl,
+                ApiKey = apiKey,
+                AccountId = relayAccountId,
+                AccountName = relayAccountName,
+                Username = relayUserEmail
+            };
+
+            var sut = new EmailSender(Options.Create(configuration), null);
```

Comment:
`BigQueryTest : IClassFixture<WebApplicationFactory<Startup>>` is for testing your BigQuery API from outside.  I mean, BigQuery API is the SUT (_system under test_) in this class.

In this test in particular, the SUT is the _EmailSender_, so, in my opinion, you should move it to another file.

Reply:
solved, the test was removed from bigquery test and was added to EmailServiceTest file

Reply:
Great, remember to remove the old code from `BigQueryTest.cs`